.github/workflows/debezium-workflow.yml

```
@@ -316,7 +319,7 @@ jobs:
 
       - name: Build Debezium Connector MySQL
         run: >
-          ./mvnw clean install -B -pl debezium-connector-mysql -am -Passembly
+          ./mvnw clean install -B -pl debezium-connector-mysql -am -P${{ matrix.profile }}
```

```suggestion
          ./mvnw clean install -B -pl debezium-connector-mysql -am -Passembly,${{ matrix.profile }}
```

Just like that @Naros?